File: src/main/java/io/openfuture/openmessanger/web/response/UserMessageResponse.java
package io.openfuture.openmessanger.web.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserMessageResponse {
    private String uniqueId;
    private String recipientName;
    private String recipientAvatarUrl;
    private String lastMessage;
    private LocalDateTime lastAt;
}

File: src/main/kotlin/io/openfuture/openmessanger/web/response/UserMessageResponse.kt
package io.openfuture.openmessanger.web.response

import java.time.LocalDateTime

data class UserMessageResponse(
    var uniqueId: String? = null,
    var recipientName: String? = null,
    var recipientAvatarUrl: String? = null,
    var lastMessage: String? = null,
    var lastAt: LocalDateTime? = null
)